examples/huggingface/pytorch/text-generation/quantization/README.md

```diff
@@ -17,15 +17,15 @@ WORK_DIR=$PWD
 # GCC 12.3 is required, please set it firstly
 # Create environment (conda recommended)
 conda create -n llm python=3.9 -y
-# install deps
+# install deps, please try gcc, gxx 12.2 if 12.3 doesn't find from conda
 conda install gcc=12.3 gxx=12.3 cxx-compiler -c conda-forge -y
 conda install cmake ninja mkl mkl-include -y
 conda install gperftools -c conda-forge -y
 
 # Install PyTorch
-python -m pip install torch==2.1.0.dev20230711+cpu torchvision==0.16.0.dev20230711+cpu torchaudio==2.1.0.dev20230711+cpu --index-url https://download.pytorch.org/whl/nightly/cpu
+python -m pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/cpu
```

Comment:
Is nightly stable? it's also dynamic. Better to use a fixed stable version.

Reply:
change another 0711 link to replace.